src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderPostProcessor.java

```diff
@@ -116,10 +122,27 @@ public CallbackResponse notifyPartiesAndPrintOrder(CallbackParams callbackParams
         Claim claim = caseDetailsConverter.extractClaim(caseDetails);
         CCDCase ccdCase = caseDetailsConverter.extractCCDCase(caseDetails);
         notifyParties(claim);
+        raiseAppInsightEvents(CaseEvent.fromValue(callbackParams.getRequest().getEventId()), ccdCase);
         String authorisation = callbackParams.getParams().get(BEARER_TOKEN).toString();
         return printOrder(authorisation, claim, ccdCase);
     }
 
+    private void raiseAppInsightEvents(CaseEvent caseEvent, CCDCase ccdCase) {
```

Comment:
It only raises one event per invocation
```suggestion
    private void raiseAppInsightEvent(CaseEvent caseEvent, CCDCase ccdCase) {
```